### PR TITLE
Star rating

### PR DESCRIPTION
**Pull request by VedanthB**
- added start rating component classes

https://user-images.githubusercontent.com/75097551/152438790-994223c7-18e2-4909-a9ea-a8ea559472dc.mov

### CODE REVIEW

File: components/star-rating.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+.rating:not(:checked) > label:hover, /* hover current star */
+.rating:not(:checked) > label:hover ~ label {
+  color: #ffd700;
+} /* hover previous stars in list */
+
+.rating > input:checked + label:hover, /* hover current star when changing rating */
+.rating > input:checked ~ label:hover,
+.rating > label:hover ~ input:checked ~ label, /* lighten current selection */
+.rating > input:checked ~ label:hover ~ label {
+  color: #ffed85;
```
Comment (krishanu2911) [inline components/star-rating.css:47]: just a suggestion to use CSS variable

File: components/star-rating.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  content: "\f005";
+}
+
+.rating > .half:before {
+  content: "\f089";
+  position: absolute;
+}
+
+.rating > label {
+  color: #ddd;
```
Comment (krishanu2911) [inline components/star-rating.css:33]: same here

**Review by krishanu2911 — COMMENTED** [review]
Added few suggestions

**Review by ashishnnnnn — COMMENTED** [review]
Code seems fine .. Good to go :+1:

**Pull request merged**